website/docs/docs/get-started-dbt.md

```diff
@@ -76,13 +76,23 @@ Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-feature
 
 [dbt Core](/docs/core/about-core-setup) is a command-line [open-source tool](https://github.com/dbt-labs/dbt-core) that enables data practitioners to transform data using analytics engineering best practices. It suits individuals and small technical teams who prefer manual setup and customization, supports community adapters, and open-source standards.
 
-Refer to the following quickstarts to get started with dbt Core:
+<div className="grid--3-col">
+
+<Card
+    title="dbt Core from a manual install"
+    body="Learn how to install dbt Core and set up a project."
+    link="https://docs.getdbt.com/guides/manual-install"
+    icon="dbt-bit"/>
 
-- [dbt Core from a manual install](/guides/manual-install) to learn how to install dbt Core and set up a project.
-- [dbt Core using GitHub Codespace](/guides/codespace?step=1) to learn how to create a codespace and execute the `dbt build` command.
+<Card
+    title="dbt Core using GitHub Codespace"
+    body="Learn how to create a codespace and execute the `dbt build` command."
```

Comment:
Hiya @mirnawong1 

I'm not too sure why but the code block hasn't populated for `dbt build` on the preview.

Kind Regards